modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -51,13 +54,10 @@ inline string reverse(string str){
 
 [CAUTION]
 ====
-If any code in `ExprFunctions.hpp` or `ExprUtil.hpp` causes a compilation error, GSQL cannot install any GSQL query, even if the GSQL query doesn't call any query UDF. Therefore, please test each new query UDF after adding it. One way of testing a function is to create a new file `test.cpp` and compile it: +
-`> g{pp} test.cpp
+If any code in `ExprFunctions.hpp` or `ExprUtil.hpp` causes a compilation error, GSQL will be unable to install _any_ new queries, whether user-defined or not. Therefore, please test each new query UDF after adding it. One way of testing a function is to make a new file `test.cpp` and compile it.
```

Comment:
This is a little tricky now. Since users can use accumulators in their UDF, those accumulator classes rely on the accumulator definition in GSQL. If they just create a `test.cpp` and put their functions in it, and those functions happen to contain an accumulator, then it actually won't compile. 

I can't think of another quick way of testing it - so perhaps we'll refrain from making a suggestion, but just keep it as a warning.